### PR TITLE
Basic Routes, Mongo creation question functionality, Socket.IO room functionality

### PR DESCRIPTION
**Pull request by joshua-j-hong**

### CODE REVIEW

File: backend/index.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+
+// Kahoot
+var indexRouter = require('./routes/index');
```
Comment (StoneT2000) [inline backend/index.js]: use const!

File: backend/index.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+const app = express();
+
+
+// Set static folder
+app.use(express.static(path.join(__dirname, 'public')));
+
+require('dotenv').config();
+var name = process.env.NAME;
+var dbpswd = process.env.DB_PSW;
+var dbname = process.env.DB_NAME;
```
Comment (StoneT2000) [inline backend/index.js]: definitely use `const`

File: backend/utils/users.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+const users = [];
+
+// Join user to chat
+function userJoin(id, username, room) {
```
Comment (StoneT2000) [inline backend/utils/users.js]: recommend to use modern `const userJoin = () => {}` syntax

**Comment by joshua-j-hong** [comment]
Fixed function declarations. Changed how room + player data is stored so now if someone tries to make a new room that already exists + has a current admin, they get an error. If the admin of a room leaves, then another admin can join the room and basically take over. Also, if everyone leaves the room, then the room is "deleted" and new players will get an error if they try to join the ID.

**Comment by anh65498** [comment]
Please, next time, write a description of what your PR can do at the beginning of the conversation so I know what to check and don't miss anything that you want your PR to do

**Review by anh65498 — APPROVED** [review]

**Pull request merged**